pygmt/helpers/caching.py

```diff
@@ -22,6 +22,7 @@ def cache_data():
         "@earth_mag_01d_g",
         "@earth_mag4km_01d_g",
         "@earth_mask_01d_g",
+        "@earth_mdt_01_g",
```

Comment:
```suggestion
        "@earth_mdt_01d_g",
```

Reply:
Fixed in commits https://github.com/GenericMappingTools/pygmt/pull/3718/commits/f7b660764cbc90d6c3d51194ceeadca8b91fb2ab and https://github.com/GenericMappingTools/pygmt/pull/3717/commits/e7180078ee018020d76a3819f4b360601f1b635c.